docs/api/hosting-api.md

```diff
@@ -0,0 +1,17 @@
+# Hosting your API with OneBlink
+
+## Deploying your API
+
+Now we are at the fun stage, deploying our newly created API.
+
+If everything is still setup from the past few steps, then we will be good to go. If not, you will have to ensure that you have logged into the OneBlink CLI and that you are in the right folder in your terminal.
+
+Now we can run the command
+
+`oneblink api deploy`
+
+After running this, you will get text in your terminal if it is okay to deploy to that environment, most likely named 'dev'. If that is fine, you can hit `y` and enter, then you're good to go. Go check the OneBlink Hosted API page and give it a refresh if required. You should see your brand new shiny API!
+
+![An image of the OneBlink Console showing a hosted API](../pics/ShinyNewAPI.png)
+
+Now, you have learnt the steps required to writing your own code and hosting your own API! Feel free to view some of the examples for code that you can use on your own Form!
```

Comment:
Might be useful to also include how to setup, configure and test the deployed options set via the Console and then show how to use the logs and metrics options for the api